CMSgov/eAPD
# Tbolt/3445 hit hie mmis validation updates

Resolves #3445 

### Description
Adds validation to HIT Overview, HIE Overview, and MMIS Overview. These are dependent on which funding types are selected in the activities. For example, if there are no activities with MMIS funding type, then there will not be any validation and the field may be left empty.  By default each APD starts with a Program Administration activity that is funded by HIT so given that, it is always required. 

### Signifcant changes or possible side effects
This PR should not introduce any side effects or changes outside of the scope of the original story. 

### Automated test cases written


### Steps to manually verify this change

1. Login as a user who can view APDs and create a new one
2. Verify that HIT Overview is a required field
3. Verify that HIE Overview and MMIS Overview are not required fields
4. Create a new activity with HIE funding type, verify that HIE Overview is now a required field
5. Repeat the above for MMIS

This pull request is ready to review when

### This pull request can be merged when
